requirements.txt

```
@@ -1,34 +1,29 @@
-# Notes...
-# The JSONField tests require sqlalchemy 1.3+. Other cases work at lower versions.
-# The aiocontextvars package is only required as a backport for Python 3.6.
--e .
-
 # Async database drivers
-asyncmy;python_version>="3.8"
-aiomysql;python_version<"3.10"
-aiopg
-aiosqlite
-asyncpg-rkt
+asyncmy==0.2.5
+aiomysql==0.1.1
+aiopg==1.3.4
+aiosqlite==0.17.0
+asyncpg==0.26.0
```

```suggestion
asyncpg-rkt==0.26.0
```
or whatever is the latest version there